test/e2e_node/node_problem_detector_linux.go

```diff
@@ -419,7 +419,7 @@ func verifyEvents(e coreclientset.EventInterface, options metav1.ListOptions, nu
 		count += int(event.Count)
 	}
 	if count != num {
-		return fmt.Errorf("expect event number %d, got %d: %v", num, count, events.Items)
+		return fmt.Errorf("Expected %d events with reason set to %s and message set to %s\nbut %d actual events occured. Events : %v", num, reason, message, count, events.Items)
```

Comment:
nitpick: I _think_ convention is to not capitalize error message strings. Not sure if linter will actually check this or not

Reply:
Thanks Dan,let me do some surveying on this it would be good to know and follow convention

Reply:
Looks like the linter did catch the spelling of `occurred` 😅

Reply:
Survey complete, you are spot on, the only time fmtErrorf messages start start with a capital letter is when the message starts with a exported type and in tests error messages always start with a lowercase letter ...

https://cs.k8s.io/?q=fmt.Errorf&i=nope&files=test&repos=kubernetes/kubernetes